### PR TITLE
Adding the ability to add custom YAPF files.

### PR DESCRIPTION
**Pull request by kiddinn**
Issue with TS that uses 4 spaces, #52 , this is to fix that issue.

### CODE REVIEW

**Comment by kiddinn** [comment]
this PR should fix this issue

**Comment by kiddinn** [comment]
There is one tiny change that needs to be done as well, so let's not merge this in quite yet

File: l2tscaffolder/lib/mapping_helper.py [hunk 1]
```diff
@@ -100,21 +105,17 @@ def RenderTemplate(self, template_filename: str, context: dict) -> str:
 
   def SetDefaultPaths(self):
```
Comment (Onager) [inline l2tscaffolder/lib/mapping_helper.py]: Please refactor this, probably passing formatter_path and template_path to the ParserMapper initializer. There's no need to have multiple levels of initialization here, and SetTemplatePath and SetFormatterPath are only called by SetDefaultPaths (and the timesketch scaffolder, in this change).

Comment (kiddinn) [reply l2tscaffolder/lib/mapping_helper.py]: ok... so I changed this to take both formatter path and template path as part of the init function

and I also renamed the class to MapperHelper

**Comment by kiddinn** [comment]
Ready for another review round... there may be a single line change needed in the template for the the sketch analyzer, otherwise this should be ready for a review round

**Pull request merged**